tests/functional/aws-node-sdk/test/bucket/put.js

```diff
@@ -257,15 +257,23 @@ describe('PUT Bucket - AWS.S3.createBucket', () => {
         location => {
             describeSkipAWS(`bucket creation with location: ${location}`,
             () => {
-                after(() => bucketUtil.deleteOne(bucketName));
+                after(done =>
+                    bucketUtil.deleteOne(bucketName).then(() => done()).catch(() => done()));
```

Comment:
```suggestion
                    bucketUtil.deleteOne(bucketName).then(done).catch(done));
```

Reply:
The goal is is to ignore errors, because in the cause of bucket creation with cold backend, the creation fails and then the cleanup throws an error because `{}` is returned. This change is to ensure errors are ignored during the cleanup. Let me know if you see a better approach here (like being specific on the error, maybe?)

Reply:
I think this is fine for this use case, ideally if we have some tests that should ignore errors and some tests that errors should fail the test we should be breaking those tests up in two different suites. But I will approve for now.